nix/evmos-rocksdb.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
```

Comment:
you can reuse the `default.nix` in repo root, add an optional parameter and some conditional flags.

Reply:
Thanks! Will refactor this